project.clj

```diff
@@ -4,4 +4,33 @@
   :license {:name "The MIT License"
             :url "http://opensource.org/licenses/MIT"}
   :dependencies [[org.clojure/clojure "1.7.0"]
-                 [com.stuartsierra/dependency "0.2.0"]])
+                 [com.stuartsierra/dependency "0.2.0"]]
+  :profiles {:dev {:dependencies [[org.clojure/clojurescript "1.7.228"]]}}
+  :plugins [[lein-doo "0.1.7"]]
+  :cljsbuild
+  {:builds [{:id "phantom-test"
+             :source-paths ["src" "test"]
+             :compiler {:output-to  "target/cljs/phantom-test/integrant-test.js"
+                        :output-dir "target/cljs/phantom-test/out"
+                        :main integrant.test-runner
+                        :optimizations :none}}
+            {:id "nashorn-test"
```

Comment:
I'm umming and ahing about this one. I know there are some people running clojurescript in nashorn with build tools. Figwheel is built currently on component. I can imagine a scenario in which something similar might use integrant and build it within nashorn.